common/app/views/fragments/amp/stylesheets/tones.scala.html

```diff
@@ -21,7 +21,8 @@
     color: #fff;
 }
 
-.tonal--tone-comment .tonal__standfirst {
+.tonal--tone-comment .tonal__standfirst,
```

Comment:
Ah right, I'm confused now, as all of the colour/tonal specific css is through nested selectors. Shall we have a chat when your back upstairs?